src/dsp/ethereum/index.ts

```diff
@@ -9,39 +11,42 @@ import { prepareBlocksByYaml } from './prepare_blocks'
 
 export { EthereumDataPrep } from './blockprep'
 
-export interface EthereumDataSourcePluginPrepareParams {
+export type EthereumDataSourcePluginPrepareParams = EthereumDSPPrepareParams
```

Comment:
this rename might not be necessary

Reply:
This modification comes from the third point of your suggestion.
> The term "DataSourcePlugin" within each subclass of DSP can be abbreviated as DSP.